timemachine/fe/free_energy.py

```diff
@@ -392,12 +428,65 @@ def _combine(self, ligand_values, host_values=None):
         return np.concatenate([host_values, ligand_values])
 
 
-def get_context(initial_state: InitialState) -> Context:
-    bound_impls = [p.to_gpu(np.float32).bound_impl for p in initial_state.potentials]
+def get_water_sampler_params(initial_state: InitialState) -> NDArray:
+    """Given an initial state, return the parameters that define the parameters of water-environment parameters
```

Comment:
Detail what environment here means, environment being both ligand + protein?

Reply:
Yes, will clarify

Reply:
Removed that line, think the following sentence does a better job explaining it: https://github.com/proteneer/timemachine/pull/1286/commits/f940b6ad80ff6ccf492b05bb76c0167d791a1d09